src/components/common/RichText.tsx

```diff
@@ -55,19 +55,18 @@ type RichTextImageProps = {
 
 const StyledRichText = styled.div`
   .responsive-object {
-    position: relative;
+    padding-bottom: 0 !important;
+    width: 50%;
   }
-
-  .responsive-object iframe,
-  .responsive-object object,
-  .responsive-object embed {
-    position: absolute;
-    top: 0;
-    left: 0;
+  .responsive-object iframe {
+    aspect-ratio: 16 / 9;
     width: 100%;
     height: 100%;
   }
-`;
+ @media (max-width: ${(props) => props.theme.breakpointMd}) {
+    .responsive-object{
+      width: 100%;
+    }`;
```

Comment:
Rather than the width always being 50% of the container we could set a maximum width, that way it won't get too small when the screen is only a little bigger than `breakpointMd`